pkg/keccak/Makefile

```diff
@@ -0,0 +1,38 @@
+PKG_NAME	:= keccak
+PKG_URL		:= https://github.com/adrianghc/KeccakCodePackage
```

Comment:
This must be changed to the original KeccakCodePackage gihbub repo.
If patches are necessary, they need to be submitted and reviewed here too.

Reply:
Yes, you are right about that. Patches are necessary, which I will push together with changes to the Makefile.